src/Avalonia.Controls/TextBox.cs

```diff
@@ -59,6 +59,9 @@ public class TextBox : TemplatedControl, UndoRedoHelper<TextBox.UndoRedoState>.I
                 o => o.SelectionEnd,
                 (o, v) => o.SelectionEnd = v);
 
+        public static readonly StyledProperty<TextBox, uint?> MaxLengthProperty =
+            AvaloniaProperty.Register<TextBox, uint>(nameof(MaxLength), defaultValue: null);
```

Comment:
Should the `Register` use `uint?` 

```suggestion
            AvaloniaProperty.Register<TextBox, uint?>(nameof(MaxLength), defaultValue: null);
```